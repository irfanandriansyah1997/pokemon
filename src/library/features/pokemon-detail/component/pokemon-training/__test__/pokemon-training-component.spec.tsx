import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import PokemonTraining from '@/library/features/pokemon-detail/component/pokemon-training';

describe(`Testing Pokemon Training Card Component`, () => {
  it(`Testing Snapshot Test`, () => {
    const ui = shallow(<PokemonTraining title="Sample Title" value="50%" />);

    expect(toJson(ui)).toMatchSnapshot();
  });
});
