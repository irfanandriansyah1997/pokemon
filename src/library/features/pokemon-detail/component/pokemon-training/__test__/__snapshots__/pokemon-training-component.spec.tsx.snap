// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Pokemon Training Card Component Testing Snapshot Test 1`] = `
<Styled(div)>
  <img
    alt="Training"
    loading="lazy"
    src="pokemon.svg"
  />
  <Styled(p)
    fontSize="normal"
    fontWeight={400}
  >
    Sample Title
  </Styled(p)>
  <Styled(p)
    color="primary"
    fontSize="medium"
    fontWeight={700}
  >
    50%
  </Styled(p)>
</Styled(div)>
`;
